redcap_bridge/test_redcap/test_server_elab_interface.py

```diff
@@ -20,12 +18,16 @@ def test_upload_template_from_csv(initialize_test_dir):
     csv_file = test_directory / 'testfiles' / 'metadata.csv'
 
     res, http_stat_code = upload_template_from_csv(server_config_json=SERVER_CONFIG_YAML, csv_file=csv_file,
-                                             title='General Template')
+                                                   title='General Template')
 
     assert http_stat_code == 201
 
 
 def test_download_experiment(initialize_test_dir):
-    res, http_stat_code = download_experiment(server_config_json=SERVER_CONFIG_YAML, experiment_id=232)
+    csv_file = test_directory / 'testfiles' / 'elabConversion' / 'download_to_csv.csv'
+    http_stat_code, df = download_experiment(save_to=csv_file, server_config_json=SERVER_CONFIG_YAML, experiment_id=232,
```

Comment:
Here you are assuming that there is already a specific experiment on the server with experiment_id 232. This might be the case for our current server, but is not the case for any elab server. Could you rewrite the test in a way that it first uploads an experiment and then downloads it again, so you can compare the content you uploaded vs the content you donwloaded? See also the approach in the corresponding redcap test: https://github.com/INT-NIT/DigLabTools/blob/8fe6bece2d622a79446b4229e33e2bf0d7dc1e53/redcap_bridge/test_redcap/test_server_interface.py#L75

Reply:
Yes, i know that test are not correct, i will see the test_download_records to see the approach and do better tests